server.go

```diff
@@ -293,6 +300,7 @@ func (s *ServerImpl) ListDeploys() ([]*Deploy, error) {
 	knownRunningDeploys := []*Deploy{}
 	deployIds := s.readDeployIdsFromDisk()
 	knownDeploys := []*Deploy{}
+	setPort, _ := s.getPortMarkedAsSet()
```

Comment:
It's not a good pattern in go to simply ignore errors. Given we wrote getPortMarkedAsSet - if these are really errors we don't care about, then change this method to log this information instead. At least that way it's not lost.